README.md

```diff
@@ -36,10 +38,17 @@ If you prefer manual installation, download, extract and run the GlareDB binary
 
 ## Getting started
 
-Start GlareDB with the `local` subcommand to start a local SQL session:
+To start a local session, run:
 
 ```shell
-./glaredb local
+./glaredb
+```
+
+If you need to pass options, you must specify `local`. To see all options for
+running locally, use `--help`:
```

Comment:
Shouldn't be needed, local args are flattened to the top level:

```
sean@Seans-Air glaredb % ./target/debug/glaredb --help          
CLI for GlareDB

Usage: glaredb [OPTIONS] [COMMAND]

Commands:
  local   Starts a local version of GlareDB (default)
  server  Starts the sql server portion of GlareDB
  help    Print this message or the help of the given subcommand(s)

Options:
  -v, --verbose...
          Log verbosity

      --json-logging
          Output logs in json format

  -q, --query <QUERY>
          Execute a query, exiting upon completion.
          
          Multiple statements may be provided, and results will be printed out one after another.

  -m, --metastore-addr <METASTORE_ADDR>
          Address to the Metastore.
          
          If not provided, an in-process metastore will be started.

      --spill-path <SPILL_PATH>
          Path to spill temporary files to

  -f, --data-dir <DATA_DIR>
          Optional file path for persisting data.
          
          Catalog data and user data will be stored in this directory.

      --mode <MODE>
          Display output mode
          
          [default: table]
          [possible values: table, json, ndjson, csv]

      --width <WIDTH>
          Max width for tables to display

      --max-rows <MAX_ROWS>
          Max number of rows to display

      --max-columns <MAX_COLUMNS>
          Max number of columns to display

  -h, --help
          Print help (see a summary with '-h')

  -V, --version
          Print version
```

E.g. I can provide `--data-dir` without specifying `local`:

```
sean@Seans-Air glaredb % ./target/debug/glaredb 
GlareDB (v0.3.0)
Type \help for help.
Using in-memory catalog
> \q
Error: Unable to handle client command: \q
> \quit
sean@Seans-Air glaredb % ./target/debug/glaredb --data-dir hello
GlareDB (v0.3.0)
Type \help for help.
Persisting database at path: hello
> create table hello (a int);
create table
> insert into hello values (1);
write success
> \quit
sean@Seans-Air glaredb % ./target/debug/glaredb --data-dir hello
GlareDB (v0.3.0)
Type \help for help.
Persisting database at path: hello
> select * from hello;
┌───────┐
│ a     │
│ ──    │
│ Int32 │
╞═══════╡
│ 1     │
└───────┘
```